pkg/stores/user/db.go

```diff
@@ -849,6 +849,42 @@ func (s *DBStore) ResetStatsAndRatings(ctx context.Context, uid string) error {
 	return nil
 }
 
+func (s *DBStore) ResetPersonalInfo(ctx context.Context, uuid string) error {
+	u := &User{}
+	p := &profile{}
+
+	if result := s.db.Where("uuid = ?", uuid).First(u); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(u).Related(p); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(p).Update("first_name", ""); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(p).Update("last_name", ""); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(p).Update("about", ""); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(p).Update("title", ""); result.Error != nil {
+		return result.Error
+	}
+	if result := s.db.Model(p).Update("avatar_url", ""); result.Error != nil {
+		return result.Error
+	}
+	return s.db.Model(p).Update("country_code", "").Error
```

Comment:
can we make this all one single Update query?

Reply:
perhaps

Reply:
yes